setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+with open("Readme.md", "r") as readme_file:
+    readme = readme_file.read()
+
+setup(
+    name="converter",
+    version="1.0.1",
```

Comment:
@MariaMaximova, from one hand it's ok to add plain version to setup.py file. From the other, I think I will suggest you adding a file `version.txt` and loading version from it as you load readme from Readme.md file.
When you will update version, you will need update only version.txt file.

Reply:
Also, I would suggest you working not directly in `dev` branch but in separate feature branch, for example, for this MR you could create a new branch `issue-1-fix-add-setup-file` or `fix-issues-1-and-2`. Then create MR from that branch to `dev` branch (not changing the version). And then you can create release MR from `dev` to `main` branch. That's like a little simplified [github workflow](https://quangnguyennd.medium.com/git-flow-vs-github-flow-620c922b2cbd) when you combine master and release branches together.

Reply:
Added in [setup.py](https://github.com/MariaMaximova/convert_temp/commit/67f37cda14e8921dad8a4e3e96f9f8abb4ad93d8)